test/gui/test_b0shim_tab.py

```diff
@@ -21,7 +21,7 @@ def test_st_plugin_b0shim_dyn_lsq_mse():
     options = {'optimizer-method': 'Least Squares',
                'optimizer-criteria': 'Mean Squared Error',
                'slices': 'Auto detect',
-               'scanner-coil-order': '1',
+               'scanner-coil-order': '0,1,2',
```

Comment:
Does this do something now that we use the checkboxes?

Reply:
It's just to test all checkboxes

Reply:
Yes but the option `scanner-coil-order` is not being used anymore. We hardcoded to click on all checkboxes in the test.

Reply:
maybe keep it to indicate what is going on in the test but add a comment to indicate that this parameter is not used.